pkg/bgp/extended-community.go

```diff
@@ -48,14 +48,23 @@ func (ext *ExtCommunity) String() string {
 		prefix = "sas="
 	case 0x0a:
 		prefix = "l2i="
+	case 0x0b:
+		prefix = "color="
```

Comment:
Just a question. Is there any convention/abbreviation which should be used for identification of type of communities?

Reply:
no official ones that I know, but would be good practice to use those displayed in XR as ref